module/VuFind/src/VuFind/Db/Service/UserService.php

```diff
@@ -43,7 +45,10 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  */
-class UserService extends AbstractDbService implements LoggerAwareInterface, DbServiceAwareInterface
+class UserService extends AbstractDbService implements
+    DbServiceAwareInterface,
```

Comment:
As above: is this needed? (I imagine it might be in future, but I don't think it is at present).